### ISSUE TITLE
filtering

### ISSUE THREAD

**Issue by gsaslis** [cytechmobile/radicle-planning-boards#74]
Copy-pasting from zulip discussion: 

maninak: For documentation puproses o τρόπος που κάνεις HATEOAS είναι πχ. στ παράδειγμα με τα filters:

user submits a filter
ui filter submit handler uses the app's router to update the route denoting the new state of the app
the router auto-updates the browser's URL to match the new route we've set
the router, which is also listening to URL changes, will pick up the "event" that the URL got changed
the router will handle the URL change, matching it to a different vue component to render and parsing URL params , e.g. the filters set, which it propagates downwards to perhaps a page vue component and eventually throughout the whole vue app. Some of those components may be set to make httpd request(s) on URL param changes furthering the reactivity cycle.
Note that the browsing session could also start straight from step 4 if the user visits (maybe even after an edit) that URL with the filters as a route and/or URL/query params.

TL;DR

we save the state on the URL
URL (aka state) changes drive (internal app state and eventually) UI changes

maninak: This is a classic case of great product design that also ends up in great DX. Handling state in HATEOAS apps is such a joy compared to ones without this approach.
